addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/__tests__/aws-accounts-service.test.js

```diff
@@ -154,6 +154,15 @@ describe('AwsAccountService', () => {
         s3Prefix: 'dummyKey',
       });
 
+      service._settings = {
```

Comment:
should add test cases to cover partition `aws-cn` as well?

Reply:
This settings is a config, I think it is fine to check the result string value is from config.